src/xz/util.c

```diff
@@ -142,6 +146,13 @@ round_up_to_mib(uint64_t n)
 }
 
 
+#ifdef _MSC_VER
+#pragma warning(push)
+#pragma warning(disable: 4474) // 'snprintf' : too many arguments passed for format string
```

Comment:
I changed the code in the master branch so that these pragmas shouldn't be needed anymore.

Reply:
Fixed in a new commit. `pragma warning` has been removed.